sdks/builds/llvm.mk

```diff
@@ -138,10 +138,22 @@ endif
 #  $(3): mxe
 define LLVMMxeTemplate
 
+# need these so that the LLVM libraries don't dependencies on shared libraries
+# of libgcc, libstdc++ etc. There are also -static-libmpx
+# -static-libmpxwrappers flags in some versions of GCC, but not the one in
+# mingw-w64 6.0.0 in Homebrew.
+#
+# TODO: Since we don't run autoconf here there's not really a convenient way to
+# figure out which ones work.  It would be nice not to hardcode this list.
+_llvm-$(1)_C_FLAGS=-static-libgcc -static-libstdc++ -static-libasan -static-libtsan -static-liblsan -static-libubsan
+_llvm-$(1)_CXX_FLAGS=-static-libgcc -static-libstdc++ -static-libasan -static-libtsan -static-liblsan -static-libubsan
+
 # -DCROSS_TOOLCHAIN_FLAGS_NATIVE is needed to compile the native tools (tlbgen) using the host compilers
 # -DLLVM_ENABLE_THREADS=0 is needed because mxe doesn't define std::mutex etc.
 # -DLLVM_BUILD_EXECUTION_ENGINE=Off is needed because it depends on threads
 _llvm-$(1)_CMAKE_ARGS = \
+	-DCMAKE_C_FLAGS=\"$$(_llvm-$(1)_C_FLAGS)\" \
```

Comment:
In a different project that has similar requirements we also added this to the cmake flags, don't know if it is needed when you add the C/CCX FLAGS, just validate that the final build opt.exe, llc.exe etc doesn't have any additional dependecy on c/c++ runtimes.

-DCMAKE_EXE_LINKER_FLAGS="-static"

Reply:
In that case we only used that flag together with the -all-static when linking mono runtime and that produced a cross compiler without any additional dependecies.